src/testing/testing_runtime.ts

```diff
@@ -13,30 +13,14 @@ import { OperonWorkflow, WorkflowHandle, WorkflowParams } from "../workflow";
 import { Http2ServerRequest, Http2ServerResponse } from "http2";
 import { ServerResponse } from "http";
 import { SystemDatabase } from "../system_database";
-import { Client } from "pg";
 import { get, has } from "lodash";
 
 /**
  * Create a testing runtime. Warn: this function will drop the existing system DB and create a clean new one. Don't run tests against your production database!
  */
 export async function createTestingRuntime(userClasses: object[], configFilePath: string = operonConfigFilePath): Promise<OperonTestingRuntime> {
-  const otr = new OperonTestingRuntimeImpl();
   const [ operonConfig ] = parseConfigFile({configfile: configFilePath});
-
-  // Drop system database
-  const pgSystemClient = new Client({
-    user: operonConfig.poolConfig.user,
-    port: operonConfig.poolConfig.port,
-    host: operonConfig.poolConfig.host,
-    password: operonConfig.poolConfig.password,
-    database: operonConfig.poolConfig.database,
-  });
-  await pgSystemClient.connect();
-  await pgSystemClient.query(`DROP DATABASE IF EXISTS ${operonConfig.system_database};`);
-  await pgSystemClient.end();
-
-  // Initialize the runtime.
-  await otr.init(userClasses, operonConfig);
+  const otr = createInternalTestRuntime(userClasses, operonConfig, undefined, /*dropSystemDB=*/true)
```

Comment:
I think its a bit dangerous to allow this parameter. It's safer to keep it in the `testing_runtime` code only...

Reply:
Will update the code.